test/Grpc.IntegrationTests/IntegrationTestBase.cs

```diff
@@ -75,6 +75,10 @@ protected IHostBuilder CreateHostBuilder(Action<IDurableTaskWorkerBuilder> confi
                 {
                     b.UseGrpc(this.sidecarFixture.Channel);
                     b.RegisterDirectly();
+                    if (clientConfigure != null)
+                    {
+                        clientConfigure!(b);
```

Comment:
The `!` here is not needed since you already checked for null above.
```suggestion
                        clientConfigure(b);
```
Alternatively, you could remove the if-conditional above and just do this: `clientConfigure?.Invoke(b)` - which is a bit more common in this repo since it's less verbose.

Reply:
Always a fan of being less verbose, updated to `?.`.